ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -622,6 +627,8 @@ open class BottomCommandingController: UIViewController, TokenizedControlInterna
         return stackView
     }()
 
+    private var headerHeight: CGFloat = 0
```

Comment:
Seems like we don't need to store this, instead could return it from `updateHeaderHeight` ? Storing might lead to some future bugs where someone else will try to use it without triggering the recalculation.